Parsers/ASimFileEvent/ProductParsers/FileEventMicrosoftSysmonFileDeleted.yaml

```diff
@@ -12,55 +12,92 @@ References:
   Link: https://aka.ms/AzSentinelFileEventDoc 
 - Title: ASIM
   Link: https://aka.ms/AzSentinelNormalization
-Description: ASIM Sysmon/Event File Deletion Event Parser (event number 23)
+Description: ASIM Sysmon/Event File Deletion Event Parser (event number 23) from "Event" and "WindowsEvent" tables.
```

Comment:
Should remove "/Event" as you did above.

Reply:
Done